lib/utils.js

```diff
@@ -597,7 +597,7 @@ exports.stackTraceFilter = function () {
 
       // Clean up cwd(absolute)
       if (/\(?.+:\d+:\d+\)?$/.test(line)) {
-        line = line.replace(cwd, '');
+        line = line.replace('(' + cwd, '(');
```

Comment:
by using `escape-string-regexp`, @ScottFreeCode meant do this:

```js
line = line.replace(require('escape-string-regexp')(cwd), '');
```

Does that not work?

Reply:
No, that doesn't.

In this case, `escape-string-regexp` change nothing on the path. (I didn't check it with `escape-string-regexp` on Windows) 
```
> var cwd = '/Users/outsideris/github/mocha/';
undefined
> require('escape-string-regexp')(cwd)
'/Users/outsideris/github/mocha/'
```

So, `line = line.replace(require('escape-string-regexp')(cwd), '');` will change `at foo (/www/Users/outsideris/github/mocha/foo/index.js:13:226)` to `at foo (/wwwfoo/index.js:13:226)`. 
It resolves nothing here and results same as before I change it.